examples/llm/benchmarks/perf.sh

```diff
@@ -0,0 +1,50 @@
+#/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+model=neuralmagic/DeepSeek-R1-Distill-Llama-70B-FP8-dynamic
+
+isl=3000
```

Comment:
maybe a comment to explain these - this represents which general use case - is it Q/A or summarization? @harryskim for confirmation

Reply:
Will revisit this on the multi-node guide PR.